index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1>My logo</h1>
+    <div id="hamburgerContainer">
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
+    </div>
+  </header>
+  <main>
+    <section id="intro">
+      <h2>
+        Hey There.
+        I’m Yuliana.
+      </h2>
+      <p id="subtitle">I'm a Software Developer</p>
+      <p id="introduction">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hesitate to contact me.</p>
+        <div id="socialMedia">
+          <figure><img src="./src/github.svg" alt="Github icon"></figure>
+          <figure><img src="./src/Linkedin icon.svg" alt="Linkedin icon"></figure>
+          <figure><img src="./src/angel list.svg" alt="Angel list icon"></figure>
+          <figure><img src="./src/twitter.svg" alt="twitter icon"></figure>
+          <figure><img src="./src/medium.svg" alt="Medium icon"></figure>
```

Comment:
- [ ] For this section, The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

-  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.